server/models/schema.sql

```diff
@@ -524,6 +524,7 @@ CREATE TABLE `employee` (
   `patient_uuid`  BINARY(16) DEFAULT NULL,
   `is_medical`    TINYINT(1) DEFAULT 0,
```

Comment:
Can we move the `is_medical` to the `title_employee` table?  It seems like `is_medical` has to do with the employee's job title, not the employee themselves, right?

From this implementation, I assume all employees have titles?  Can multiple employees have the same title?

Reply:
That's a really good idea that would make things a lot easier. I can do it in another PR